Makefile.am

```diff
@@ -638,9 +638,13 @@ tests_test_security_curve_SOURCES += \
 endif
 
 tests_test_security_curve_LDADD = \
-	src/libzmq.la
+	src/libzmq.la $(LIBUNWIND_LIBS)
```

Comment:
Sanity check : Wasn't libunwind an optional dependency (or am I mistaking projects)? Should this either become required dependency for this test to get built, or `#if HAVE_LIBUNWIND` trickery is needed?